src/Database/Adapter/MariaDB.php

```diff
@@ -142,6 +142,38 @@ public function createCollection(string $name, array $attributes = [], array $in
         return true;
     }
 
+    /**
+     * Get Collection Size
+     * @param string $collection
+     * @return int
+     * @throws DatabaseException
+     */
+    public function getSizeOfCollection(string $collection): int
+    {
+        $filteredName = $this->filter($collection);
+        $collectionName = "{$this->getNamespace()}_{$filteredName}";
+        $database = $this->getDefaultDatabase();
+        $name = $database . '/' . $collectionName;
+
+        $query = $this->getPDO()->prepare("
+             SELECT SUM(FS_BLOCK_SIZE + ALLOCATED_SIZE)  
+             FROM INFORMATION_SCHEMA.INNODB_SYS_TABLESPACES
+             WHERE NAME = CONCAT(:name)
+         ");
+
+        $query->bindParam(':name', $name);
+
+        try {
+             $query->execute();
+             $size = $query->fetchColumn();
+        }
+        catch (PDOException $e) {
```

Comment:
Catch should be on the same line as the closing brace

```suggestion
        } catch (PDOException $e) {
```

Reply:
done